apps/civicsignalblog/.eslintrc.js

```diff
@@ -6,6 +6,16 @@ module.exports = {
       webpack: {
         config: "./eslint.webpack.config.js",
       },
+      "eslint-import-resolver-custom-alias": {
+        alias: {
+          "#civicsignalblog": "./src",
+        },
+        extensions: [".js"],
+      },
+      typescript: {
+        alwaysTryTypes: false,
+        project: "./tsconfig.json",
+      },
```

Comment:
Since these are JS files, can you add a comment describing why we need these changes e.g. `alwaysTryTypes: false`?